### PR TITLE
MODCAT-216: Create an API for control fields

### PR DESCRIPTION
**Pull request by eturienzo**
## Purpose

<a href="https://issues.folio.org/browse/MODCAT-216">MODCAT-216</a> Create an API for control fields
#### TODOS and Open Questions

- Create a new API to get the control fields values for authority records. 

- This API will be used to get the diferent values for the selects that can be show in the interface.

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -310,6 +310,16 @@
...
         },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/marccat/auth-fixed-fields-code-groups",
+          "modulePermissions": [
+            "configuration.all"
+          ],
+          "permissionsRequired": []
```
Comment (craigmcnally) [inline descriptors/ModuleDescriptor-template.json:321]: no required permissions? This means that the API can be called w/o logging in... Is that really a requirement?

File: ramls/marccat.raml [hunk 2]
```diff
@@ -150,3 +150,26 @@ resourceTypes:
...
               example: "Error in validation"  
+  /auth-fixed-fields-code-groups:
+    displayName: Get authority control fields values
+    type:
+      collection-item:
+        exampleItem: !include examples/authorityFixedField.json
+        schema: authority
+    post:
+      is: [validate
+        ]
```
Comment (craigmcnally) [inline ramls/marccat.raml:161]: probably can move this up

File: src/main/java/org/folio/marccat/enumaration/CodeListsType.java [hunk 3]
```diff
@@ -781,5 +781,120 @@ public String toString() {
     public String toString() {
       return "VM_MATERIAL_CODE";
     }
+  },
+  SUBJECT_DESCRIPTOR {
+    @Override
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/enumaration/CodeListsType.java:786]: This is strange to me... Why do you need to override `toString()` like this if you're going to return the same value as the enum name?  By default `toString()` will return the name of the enum, so AFAICT this is kind of a waste.

For example:
```
  public static void main(String[] args) throws Exception {
    System.out.println(CodeListsType.WITH_TO_STRING);
    System.out.println(CodeListsType.WITH_TO_STRING.name());
    System.out.println(CodeListsType.WITH_TO_STRING.toString());
    
    System.out.println(CodeListsType.WITHOUT_TO_STRING);
    System.out.println(CodeListsType.WITHOUT_TO_STRING.name());
    System.out.println(CodeListsType.WITHOUT_TO_STRING.toString());
  }
```
Output:
```
WITH_TO_STRING
WITH_TO_STRING
WITH_TO_STRING
WITHOUT_TO_STRING
WITHOUT_TO_STRING
WITHOUT_TO_STRING
```
Relevant impl. snippet: 
```
public enum CodeListsType {
  WITHOUT_TO_STRING,
  WITH_TO_STRING {
    @Override
    public String toString() {
      return "WITH_TO_STRING";
    }
  },
```

File: src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java [hunk 4]
```diff
@@ -1,34 +1,47 @@
...
-    @RequestHeader(OKAPI_TENANT_HEADER_NAME) final String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl) {
+  public FixedFieldCodesGroup getFixedFieldCodesGroupsByLeader(@RequestParam final String leader,
+      @RequestParam(name = "code", defaultValue = Global.MATERIAL_TAG_CODE) final String code,
+      @RequestParam final String lang, @RequestParam(required = false) final String valueField,
+      @RequestHeader(OKAPI_TENANT_HEADER_NAME) final String tenant, @RequestHeader(Global.OKAPI_URL) String okapiUrl) {
     return doGet((storageService, configuration) -> {
       final FixedFieldCodesGroup fixedFieldCodesGroup = new FixedFieldCodesGroup();
-      final Map<String, Object> map = storageService.getMaterialTypeInfosByLeaderValues(leader.charAt(6), leader.charAt(7), code);
+      final Map<String, Object> map = storageService.getMaterialTypeInfosByLeaderValues(leader.charAt(6),
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java:43]: Is this safe?  Do we know that `leader` is long enough, or could this result in an IndexOutOfBoundsException?

File: src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java [hunk 5]
```diff
@@ -38,48 +51,71 @@ public FixedFieldCodesGroup getFixedFieldCodesGroupsByLeader(
...
-  public FixedFieldCodesGroup getFixedFieldCodesGroups(
-    @RequestParam(required = false) final String leader,
-    @RequestParam final String code,
-    @RequestParam final int headerTypeCode,
-    @RequestParam final String lang,
-    @RequestParam(required = false) final String valueField,
-    @RequestHeader(OKAPI_TENANT_HEADER_NAME) final String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl) {
+  public FixedFieldCodesGroup getFixedFieldCodesGroups(@RequestParam(required = false) final String leader,
+      @RequestParam final String code, @RequestParam final int headerTypeCode, @RequestParam final String lang,
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java:62]: I personally find it much easier to read the old format w/ one annotation per line.

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java [hunk 6]
```diff
@@ -1,39 +1,57 @@
...
-  "levelOfCompressionCode", "reformattingQualityCode", "productionDetailsCode", "reductionRatioRangeCode", "reductionRatioCode", "emulsionOnFilmCode",
-  "presentationFormatCode", "productionElementsCode", "refinedCategoriesOfColorCode", "kindOfColorStockCode", "deteriorationStageCode", "completenessCode",
-  "inspectionDate", "primarySupportMaterialCode", "baseOfEmulsionCode", "videoRecordingFormatCode", "remoteSensingDataTypeCode", "soundOnMediumOrSeparateCode", "altitudeOfSensorCode", "attitudeOfSensorCode",
-  "cloudCoverCode", "platformConstructionTypeCode", "platformUseCode", "sensorTypeCode", "remoteDataTypeCode", "speedCode", "grooveWidthCode",
-  "tapeWidthCode", "tapeConfigurationCode", "discTypeCode", "sndMaterialTypeCode", "cuttingTypeCode", "specialPlaybackCharacteristicsCode",
-  "storageTechniqueCode", "classOfBrailleWritingCode1", "classOfBrailleWritingCode2", "levelOfContractionCode", "brailleMusicFormatCode1",
-  "brailleMusicFormatCode2", "brailleMusicFormatCode3", "specificPhysicalCharacteristicsCode",
-  "formatCode", "sequenceNumber", "attributes"})
+@JsonPropertyOrder({ "keyNumber", "categoryCode", "description", "headerTypeCode", "code", "displayValue",
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/resources/domain/FixedField.java:20]: just curious... is it really important to have json properties in a particular order?

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java [hunk 7]
```diff
@@ -448,10 +465,48 @@
   @JsonProperty("formatCode")
   private String formatCode;
 
+  @JsonProperty("subjectDescriptor")
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/resources/domain/FixedField.java:468]: Aren't these annotations superfluous if using the same value as the member name?

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java [hunk 8]
```diff
@@ -2882,8 +2940,7 @@ public FixedField withBaseOfEmulsionCode(String baseOfEmulsionCode) {
     return this;
   }
 
-
-  //DA QUI
+  // DA QUI
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/resources/domain/FixedField.java:2943]: ?

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java [hunk 9]
```diff
@@ -3577,14 +3957,8 @@ public void setSequenceNumber(int sequenceNumber) {
...
-    MUSIC("music"),
-    MAP("map"),
-    COMPUTER_FILE("computerFile"),
-    BOOK("book"),
-    MIXED_MATERIAL("mixedMaterial"),
-    CONTINUING_RESOURCE("continuingResource"),
-    VISUAL_MATERIAL("visualMaterial"),
-    UNDEFINED("undefined");
+    MUSIC("music"), MAP("map"), COMPUTER_FILE("computerFile"), BOOK("book"), MIXED_MATERIAL("mixedMaterial"),
+    CONTINUING_RESOURCE("continuingResource"), VISUAL_MATERIAL("visualMaterial"), UNDEFINED("undefined");
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/resources/domain/FixedField.java:3961]: Again, the old formatting is much easier to read IMO

File: src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java [hunk 10]
```diff
@@ -1,21 +1,64 @@
...
+import static org.folio.marccat.config.constants.Global.SERIAL_TYPE;
+import static org.folio.marccat.config.constants.Global.SOUND_ON_MEDIUM_OR_SEPARATE_CODE;
+import static org.folio.marccat.config.constants.Global.SOUND_RECORDING;
+import static org.folio.marccat.config.constants.Global.SPECIFIC_MATERIAL_DESIGNAION_ON_CODE;
+import static org.folio.marccat.config.constants.Global.TACTILE_MATERIAL;
+import static org.folio.marccat.config.constants.Global.TEXT_CODE;
+import static org.folio.marccat.config.constants.Global.UNSPECIFIED;
+import static org.folio.marccat.config.constants.Global.VIDEO_RECORDING;
+import static org.folio.marccat.config.constants.Global.VISUAL_TYPE;
+import static org.folio.marccat.enumaration.CodeListsType.*;
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java:44]: It's generally frowned upon to use wildcard includes.

File: src/main/java/org/folio/marccat/dao/persistence/T_AUT_BLNGL_USG.java [hunk 11]
```diff
@@ -0,0 +1,9 @@
+package org.folio.marccat.dao.persistence;
+
+/**
+ * @author elena
+ *
+ */
+public class T_AUT_BLNGL_USG extends T_SINGLE_CHAR {
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/dao/persistence/T_AUT_BLNGL_USG.java:7]: Can you please explain the purpose of these empty classes?  Also, is there a reason they need to be named like this?  FOLIO has adopted the pattern of naming classes with CamelCase.  See sonarqube.

File: src/main/java/org/folio/marccat/integration/AuthorityStorageService.java [hunk 12]
```diff
@@ -51,6 +58,47 @@ public void setStorageService(StorageService storageService) {
...
+    final CodeTableDAO dao = new CodeTableDAO();
+    return dao.getList(getStorageService().getSession(), T_AUT_ENCDG_LVL.class, locale(lang));
+  }
+
+  /**
+   * Returns the codes list associated with the given language and key.
+   *
+   * @param lang         the language code, used here as a filter criterion.
+   * @param codeListType the code list type key.
+   * @return a list of code / description tuples representing the date type
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/integration/AuthorityStorageService.java:92]: Typo?  `date` -> `data` ?

File: src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java [hunk 13]
```diff
@@ -115,15 +183,24 @@ public static void setPhysicalInfoSNDcodes(final String lang, final StorageServi
...
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("attitudeOfSensorCode", storageService.getCodesList(lang, RSI_ATTITUDE).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("cloudCoverCode", storageService.getCodesList(lang, RSI_CLOUD_COVER).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("platformConstructionTypeCode", storageService.getCodesList(lang, RSI_PLAT_CONSTRUCTION).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("platformUseCode", storageService.getCodesList(lang, RSI_PLAT_USE).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("sensorTypeCode", storageService.getCodesList(lang, RSI_SENSOR_TYPE).stream().map(toPairItem).collect(toList())));
-    fixedFieldCodesGroup.addResults(new FixedFieldElement("remoteSensingDataTypeCode", storageService.getCodesList(lang, RSI_DATA_TYPE).stream().map(toPairItem).collect(toList())));
+  public static void setPhysicalInfoRSIcodes(final String lang, final StorageService storageService,
```
Comment (craigmcnally) [inline src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java:186]: AFAICT this is just reformatting... please do this in a separate PR;  or at the **very least** add a comment to the PR description indicating which changes are purely re-formatting.

File: src/main/resources/hibernate.cfg.xml [hunk 14]
```diff
@@ -130,6 +130,27 @@
 		<mapping resource="org/folio/marccat/dao/persistence/T_ITM_BIB_LVL.hbm.xml"/>
 		<mapping resource="org/folio/marccat/dao/persistence/T_NTR_OF_CNTNT.hbm.xml"/>
 		<mapping resource="org/folio/marccat/dao/persistence/T_ITM_REC_TYP.hbm.xml"/>
+    <mapping resource="org/folio/marccat/dao/persistence/T_AUT_REC_STUS.hbm.xml"/>
```
Comment (craigmcnally) [inline src/main/resources/hibernate.cfg.xml:133]: Please be consistent with tab vs spaces

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java [hunk 15]
```diff
@@ -8,31 +11,22 @@
...
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "008")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "008").param("lang", "ita").headers(headers)
```
Comment (craigmcnally) [inline src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:28]: IMO The old format is preferred

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java [hunk 16]
```diff
@@ -41,36 +35,18 @@ public void getFixedFieldsFromLeader_return200Status() {
...
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "007")
-      .param("headerTypeCode", "23")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "007").param("headerTypeCode", "23")
```
Comment (craigmcnally) [inline src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:48]: See my earlier comments about large amounts of reformatting...

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java [hunk 17]
```diff
@@ -422,57 +245,56 @@ public void getFixedFieldsOfVisualMaterial_return200Status() {
...
-      .param("headerTypeCode", "31")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "008").param("headerTypeCode", "31")
+        .param("lang", "ita").headers(headers).when().get(url).then().statusCode(200);
   }
```
Comment (craigmcnally) [inline src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:261]: is the @Test annotation omitted intentionally?

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java [hunk 18]
```diff
@@ -422,57 +245,56 @@ public void getFixedFieldsOfVisualMaterial_return200Status() {
...
+  public void getAuthorityFixedFieldsByTag008Return200Status() {
+
+    String url = getURI(AUT_FIXED_FIELDS_CODE_GROUPS_URL);
+    Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
+    given().param("leader", AUTHORITY_LEADER).param("code", "008").param("headerTypeCode", "10").param("lang", "eng")
+        .headers(headers).when().get(url).then().statusCode(200);
+  }
+
+  public void getAuthorityFixedFieldCodesGroupsFailed() {
```
Comment (craigmcnally) [inline src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:291]: is the @test annotation omitted intentionally?

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java (hunk 6 above)
Comment (carmentrazza) [reply src/main/java/org/folio/marccat/resources/domain/FixedField.java:20]: In intellij the maximum line with 120 characters. I've seen you use 80 characters per line. Can you change this setting? To have the same settings.

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java (hunk 15 above)
Comment (carmentrazza) [reply src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:28]: Please, keep the old formatting about tests and APIs.

File: src/main/java/org/folio/marccat/enumaration/CodeListsType.java (hunk 3 above)
Comment (carmentrazza) [reply src/main/java/org/folio/marccat/enumaration/CodeListsType.java:786]: I agree with Craig.

File: src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java [hunk 19]
```diff
@@ -280,17 +420,83 @@ public static void setPhysicalInfoTCTcodes(final String lang, final StorageServi
...
+   * @param fixedFieldCodesGroup the fixedFieldCodesGroup to populate.
+   * @param storageService       the storage service.
+   * @param lang                 the lang associated with the current request.
+   * @param headerTypeCode       the header type code selected.
+   * @param tag                  the tag code.
+   */
+  public static void injectAuthorityMaterialCodes(final FixedFieldCodesGroup fixedFieldCodesGroup,
+      final AuthorityStorageService authorityStorageService, final String lang) {
+
+    // fixedFieldCodesGroup.addResults(new FixedFieldElement("enteredOnFileDate",
```
Comment (carmentrazza) [inline src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java:458]: Remove commented block.

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (eturienzo) [reply descriptors/ModuleDescriptor-template.json:321]: I have changed the permissions. Sorry.

File: ramls/marccat.raml (hunk 2 above)
Comment (eturienzo) [reply ramls/marccat.raml:161]: I have changed it

File: src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java (hunk 5 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java:62]: I have changed it

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java (hunk 6 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/resources/domain/FixedField.java:20]: I have order by the 008 tag subparts. Is any particular order necessary?

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java (hunk 8 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/resources/domain/FixedField.java:2943]: I remove it. Sorry

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java (hunk 9 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/resources/domain/FixedField.java:3961]: I have changed it

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java (hunk 6 above)
Comment (carmentrazza) [reply src/main/java/org/folio/marccat/resources/domain/FixedField.java:20]: No, it's not necessary

File: src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java (hunk 4 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java:43]: It is bibliographic, @carmentrazza could you answer him, please?

File: src/main/java/org/folio/marccat/dao/persistence/T_AUT_BLNGL_USG.java (hunk 11 above)
Comment (carmentrazza) [reply src/main/java/org/folio/marccat/dao/persistence/T_AUT_BLNGL_USG.java:7]: I have a question on this point for Craig McNally. The name corresponds to that of the table. We have many tables in mod-marccat. In particular, the T_s are decoding tables. It is preferable to leave them with this name for greater readability. I saw that sonarqube reports them. Should they all be changed, what do you suggest or do you suggest to leave them like this?

File: src/main/java/org/folio/marccat/resources/domain/FixedField.java (hunk 7 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/resources/domain/FixedField.java:468]: I have removed them.

File: src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java (hunk 4 above)
Comment (carmentrazza) [reply src/main/java/org/folio/marccat/resources/FixedFieldCodesGroupAPI.java:43]: The leader is 24 characters long, there should be no problem.

File: src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java (hunk 10 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java:44]: I have changed it.

File: src/main/java/org/folio/marccat/enumaration/CodeListsType.java (hunk 3 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/enumaration/CodeListsType.java:786]: I have changed them

File: src/main/java/org/folio/marccat/integration/AuthorityStorageService.java (hunk 12 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/integration/AuthorityStorageService.java:92]: Date format.

File: src/main/resources/hibernate.cfg.xml (hunk 14 above)
Comment (eturienzo) [reply src/main/resources/hibernate.cfg.xml:133]: I have changed it

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java (hunk 16 above)
Comment (eturienzo) [reply src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:48]: I have changed it.

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java (hunk 18 above)
Comment (eturienzo) [reply src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:291]: I have forgotten. Sorry

File: src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java (hunk 19 above)
Comment (eturienzo) [reply src/main/java/org/folio/marccat/resources/shared/FixedFieldUtils.java:458]: I remove it

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java (hunk 15 above)
Comment (eturienzo) [reply src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:28]: I have changed them

File: src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java (hunk 17 above)
Comment (eturienzo) [reply src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:452]: It is a bibliographic test. @carmentrazza, could you check it, please?

Comment (carmentrazza) [reply src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:452]: It is an oversight, the  @Test annotation must be added and the error code changed from 200 to 500.
It will be done in another PR.

Comment (eturienzo) [reply src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java:452]: Ok, thanks

**Review by carmentrazza — APPROVED** [review]

**Review by LeireDiez — APPROVED** [review]
